buildSrc/src/main/java/Dependencies.kt

```diff
@@ -41,7 +41,7 @@ object Versions {
     object Test {
         const val robolectric = "4.6.1"
         const val mockito = "3.11.0"
-        const val coroutines = "1.5.0"
+        const val coroutines = "1.6.0"
```

Comment:
AC and Fenix uses the same coroutines version in app code and in tests.
Can we also use the same `Kotlin.coroutines` version in the `testing_coroutines` dependency and drop this separate property from `Test`?